src/Classes/Utils/Utils.ts

```diff
@@ -27,6 +27,40 @@ export class Utils {
         } : null;
     }
 
+    public static hslToRgb(hsl: IHsl) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature: 'hslToRgb' to have a typedef](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5098130)